spec/yoti/dynamic_share_service/extension/thirdparty_attribute_extension_spec.rb

```diff
@@ -18,7 +18,7 @@
         .build
     end
     it 'marshals into json' do
-      expected = '{"type":"THIRD_PARTY_ATTRIBUTE","content":{"expiry_date":"2006-04-03T02:30:50.000+00:00","definitions":[{"name":"AttributeName"}]}}'
+      expected = '{"type":"THIRD_PARTY_ATTRIBUTE","content":{"expiry_date":"2006-04-03T02:30:50.000+00:00","definitions":["AttributeName"]}}'
```

Comment:
This should remain `"definitions":[{"name":"AttributeName"}]` - in other SDKs each definition is a JSON serialisable object, e.g. https://github.com/getyoti/yoti-node-sdk/blob/master/src/data_type/attribute.definition.js

Reply:
I misunderstood your prior comment. Have no fixed.